### ISSUE TITLE
resizing the bounding box

### ISSUE THREAD

**Issue by haviduck** [AlexeyAB/darknet#7018]
hello,  my network is 1024 x 576. the input is 1920x1080.

im building a function where i grab a roi and present it in a separate plt.show for close study. so i return the original frame from cv2.capturevideo instead and trying to scale the bbox so my rectangle Will appear correctly in the 1920x1080 frame. 
it definitely feels like an easy thing but ive not been able to do it at all. 
my bbox is in the format of x1, y1, x2, y2 = bbox2points from the py files.  then i use:

`roi=originalframe[y1:y2, x1:x2] `

i apologize if This has been asked before, but if so i have not been able to find it. 

tldr; how do i correctly scale a bbox along with a window.
